packages/frontend/src/actions/account.js

```diff
@@ -460,7 +459,6 @@ export const finishAccountSetup = () => async (dispatch, getState) => {
     await dispatch(refreshAccount());
     await dispatch(getBalance());
     await dispatch(staking.clearState());
-    dispatch(tokens.clearState());
```

Comment:
Pretty sure these lines are gone as soon as we land prior PR, so this shouldn't be part of this diff

Reply:
Done.

Reply:
Looking at the current `master`, it appears that this code is actually still in the codebase now; I think this is a bug since the _handlers_ for these actions were deleted in #2002 (now these dispatch calls are no-ops) -- please create a separate fix for this problem, we doesn't need to block this PR and wouldn't want to roll a bug-fix into this anyhow :)